pkgs/desktops/xfce/applications/xfmpc/default.nix

```
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, autoreconfHook
+, xfce4-dev-tools
+, libxfce4ui
+, libxfce4util
+, pkg-config
+, vala
+, gtk3
+, glib
+, libmpd
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xfmpc";
+  # Last release is too old
+  version = "0.3.1-2024-05-29";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.xfce.org";
+    owner = "apps";
+    repo = "xfmpc";
+    rev = "cf40dffec6e9b80abb1f1aa6d7dceef4790173dc";
+    hash = "sha256-moCWSLGBJuWM4/lRJi6D3w38iJeCntLo3Vl/eVfu7lw=";
+  };
+  nativeBuildInputs = [
+    autoreconfHook
+    xfce4-dev-tools
+    pkg-config
+    vala
+    # Needed both here and in buildInputs for cross compilation to work
+    libxfce4ui
+    libxfce4util
+  ];
+  buildInputs = [
+    gtk3
+    glib
+    libxfce4ui
+    libmpd
+  ];
+  configureFlags = [
+    # From some reason, upstream conditioned this maintainer mode with the
+    # search of the vala compiler in `configure.ac`. Without `VALAC` set in the
+    # makefiles, the build fails with a wierd and hard to understand error.
+    "--enable-maintainer-mode"
+  ];
+
+  meta = with lib; {
+    description = "MPD client written in GTK";
+    homepage = "https://docs.xfce.org/apps/xfmpc/start";
+    changelog = "https://gitlab.xfce.org/apps/xfmpc/-/blob/${finalAttrs.src.rev}/NEWS";
+    license = licenses.gpl2Only;
+    maintainers = with maintainers; [ doronbehar ];
```

To also show up on my repology [profile](https://repology.org/maintainer/rjl931189261@126.com) 😂 

```suggestion
    maintainers = with maintainers; [ doronbehar ] ++ teams.xfce.members;
```

---

> To also show up on my repology

Haha, welcome to the maintainance team :).